core/src/view/view.h

```diff
@@ -107,6 +107,9 @@ class View {
     /* Gets the normal matrix; transforms surface normals from model space to camera space */
     const glm::mat3& getNormalMatrix() const { return m_normalMatrix; }
 
+    /* Returns the eye position in world space */
+    const glm::vec3& getEye() const { return m_eye; }
```

Comment:
I don't think there's a net gain to changing all usages of `View` getters right now. `View::pixelScale` and `View::pixelsPerMeter` already use the abbreviated form, so I have no strong opinion on `eye`/`getEye`.